pkg/controllers/management/k3sbasedupgrade/k3sbased_upgrade_handler.go

```diff
@@ -21,6 +19,28 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 )
 
+const (
+	// ConditionalPSPValues defines the values.yaml content passed to the app. Specifically,
+	// it denotes if PodSecurityPolicies should be deployed with the version of SUC being
+	// deployed / updated. In clusters >= 1.25 PSP's are not available, however we should
+	// continue to deploy them in sub 1.25 clusters as they are required for cluster hardening.
+	ConditionalPSPValues = `global:
+  cattle:
+    systemDefaultRegistry: ""
+    psp:
+      enabled: %t
+
+systemUpgradeController:
```

Comment:
Do we actually need to define this bit of the values file? I believe helm should use the chart values by default

Reply:
I believe we have to specify them all, not because of how helm works but because of how Rancher handles that field when templating the app

https://github.com/rancher/rancher/blob/ea94f0ad4fa168a597da1bc6df74a24faf28507c/pkg/controllers/managementuserlegacy/helm/common.go#L148-L171

https://github.com/rancher/rancher/blob/ea94f0ad4fa168a597da1bc6df74a24faf28507c/pkg/controllers/managementuserlegacy/helm/common/common.go#L289-L297

I'll double check this though, not including them all would be ideal

Reply:
I realized that the `app.spec.Answers` field eventually gets used when crafting the `--set` flag, allowing us to just specify that field instead of the entire file. PR has been tested and updated